app/src/main/java/mozilla/lockbox/presenter/SettingPresenter.kt

```diff
@@ -74,16 +74,20 @@ class SettingPresenter(
             .subscribe {
                 if (it is FingerprintAuthAction.OnAuthentication) {
                     when (it.authCallback) {
-                        is FingerprintAuthDialogFragment.AuthCallback.OnAuth -> dispatcher.dispatch(
-                            SettingAction.UnlockWithFingerprint(true)
-                        )
+                        is FingerprintAuthDialogFragment.AuthCallback.OnAuth ->
```

Comment:
please avoid committing formatting changes without any actual code changes! 🙇‍♀️

Reply:
change still not made :(